config/table_definitions.yaml

```diff
@@ -105,6 +105,20 @@ tables:
     min_year: *global_min_year
     has_cur: true
     has_parid: true
+    # Override land rates that are stored as NUMBER types with no set precision
+    # or scale, meaning they default to the maximum precision and scale that
+    # Oracle supports (38 and 127, respectively). Presto performance is best up
+    # to a precision of 18, so we set precision to that high bar and truncate
+    # scale to a high but reasonable value.
```

Comment:
See here for docs supporting this performance claim: https://prestodb.github.io/docs/current/language/types.html#fixed-precision

Reply:
**suggestion (non-blocking):** IMO let's include this link in the comment.

Reply:
Good thinking, done in bf1b3070006131051683ecf2c470d938a372323f.